templates/airflow-scc-anyuid.yaml

```
@@ -21,6 +21,7 @@ users:
 - system:serviceaccount:{{ .Release.Namespace }}:{{ .Release.Name }}-scheduler-serviceaccount
 - system:serviceaccount:{{ .Release.Namespace }}:{{ .Release.Name }}-webserver-serviceaccount
 - system:serviceaccount:{{ .Release.Namespace }}:{{ .Release.Name }}-worker-serviceaccount
+# TODO: ^^ service account names are different now, _and_ can be changed
```

```suggestion
- system:serviceaccount:{{ .Release.Namespace }}:{{ include "scheduler.serviceAccountName" . }}
- system:serviceaccount:{{ .Release.Namespace }}:{{ include "webserver.serviceAccountName" . }}
- system:serviceaccount:{{ .Release.Namespace }}:{{ include "worker.serviceAccountName" . }}
```

Should the worker serviceaccount be conditional based on executor? We don't even create it unless we are using CeleryExecutor or KubernetesExecutor.